### PR TITLE
Install stylelint & create .stylelintrc.json file

### PR DESCRIPTION
**Pull request by DNaunin**
- Use npm to install stylelint
- Create a .stylelintrc.json configuration file
- Run stylelint on all .css files

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "stylelint-config-standard"
```
Comment (fgottesmann) [inline .stylelintrc.json]:
```suggestion
  "extends": "stylelint-config-prettier"
```

**Comment by HorstFrank** [comment]
wonderfull

**Review by fgottesmann — APPROVED** [review]
You should take a look at https://github.com/prettier/stylelint-config-prettier to finish your pullrequest

**Review by phlgr — CHANGES_REQUESTED** [review]
Where is your stylelint script?

**Pull request merged**